packages/@react-spectrum/illustrated-message/src/IllustratedMessage.tsx

```diff
@@ -32,36 +32,41 @@ function IllustratedMessage(props: SpectrumIllustratedMessageProps, ref: DOMRef<
     illustrationProps,
     headingProps
   } = useIllustratedMessage(props);
+  let headingClassName = classNames(
+    {},
+    classNames(typographyStyles, 'spectrum-Heading', 'spectrum-Heading--pageTitle'),
+    classNames(styles, 'spectrum-IllustratedMessage-heading')
+  );
+  let contentClassName = classNames(
+    {},
+    classNames(typographyStyles, 'spectrum-Body--secondary'),
+    classNames(styles, 'spectrum-IllustratedMessage-description')
+  );
+
+  if (!slots) {
+    slots = {
+      container: {UNSAFE_className: styles['spectrum-IllustratedMessage-container']},
+      illustration: {UNSAFE_className: styles['spectrum-IllustratedMessage-illustration'], ...illustrationProps},
+      header: {UNSAFE_className: styles['spectrum-IllustratedMessage-header']},
+      heading: {UNSAFE_className: headingClassName, ...headingProps},
+      content: {UNSAFE_className: contentClassName}
+    };
+  }
 
-  // todo replace h2 with rsp heading when it exists
   return (
-    <div
+    <section
```

Comment:
why section?

Reply:
@majornista do you have a suggestion on what element illustrated message should use?